### DISCUSSION TITLE
Baked Lighting

### DISCUSSION THREAD

**Discussion by BookShelf1982** [ezEngine/ezEngine#936] [Q&A] [answered]
Are there any plans on integrating baked lighting into the engine or will it be just dynamic lighting?

**Comment by C-Core** [comment, marked as answer]
At some point we want to add a probe based GI solution to support dynamic global illumination. This solution will include some form of baking data into probes. Once we have that infrastructure one could e.g. bake only indirect light into probes or even bake the full lighting into the probes. It would be also possible to select that on a per-light basis. 
But it is still a long way to go and it would only be probe based. Depending on your quality needs, this might be good enough or not. There are no plans to support lightmap baking at the moment.
